hera_cal/tests/test_firstcal.py

```diff
@@ -313,3 +313,20 @@ def test_single_file_execution_nocalfile(self):
         nt.assert_true(os.path.exists(objective_file))
         os.remove(objective_file)
         return
+
+    def test_overwrite(self):
+        objective_file = os.path.join(
+            DATA_PATH, 'zen.2457698.40355.xx.HH.uvcAA.first.calfits')
+        xx_vis4real = os.path.join(DATA_PATH, xx_vis)
+        if os.path.exists(objective_file):
+            os.remove(objective_file)
+        os.system("touch {0}".format(objective_file))
```

Comment:
There is probably a more OS-independent way to achieve this. Maybe something like:
```python
open(objective_file, 'a').close()
```

Reply:
good idea